ui/apps/everest/src/api/api.ts

```diff
@@ -48,17 +49,18 @@ export const addApiErrorInterceptor = () => {
             location.replace('/login');
           }
 
+          if (
+            error.response.status === 400 &&
+            message.includes(MISSING_MALFORMED_JWT_MESSAGE)
```

Comment:
What if we get a 401? Shouldn't we also be logged out?

Reply:
@recharte  we already had a check for HTTP 401 in this file. For the sake of simplicity, I unified both in the same clause now